database/src/main/resources/migrations/ddl/V1.0.1__notifications_migration.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE notifications
+(
+    id             INT GENERATED ALWAYS AS IDENTITY,
+    user_id        INT       NOT NULL,
+    email          VARCHAR(255),
+    marked_as_read BOOLEAN DEFAULT FALSE,
+    created_at     TIMESTAMP NOT NULL,
+    payload        VARCHAR(255) NOT NULL,
+    type           VARCHAR(255),
```

Comment:
This column's too big, since we control what we insert into it.
Let's set it to 20